libs/remix-ui/workspace/src/lib/actions/index.ts

```diff
@@ -373,6 +374,33 @@ export const copyFile = async (src: string, dest: string) => {
   }
 }
 
+export const copyShareURL = async (path: string) => {
+  const fileManager = plugin.fileManager
+
+  try {
+    const host = '127.0.0.1'
+    const port = 5001
+    const protocol = 'http'
+    // const projectId = ''
```

Comment:
should you remove this?

Reply:
this will be required while connecting with actual node